G/GoogleCodeSearch/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GoogleCodeSearch"
+version = v"1.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/google/codesearch.git", "8ba29bd255b740aee4eb4e4ddb5d7ec0b4d9f23e")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p ${prefix}/share/licenses
+cp codesearch/LICENSE $prefix/share/licenses/
+mkdir gsc
+cd gsc
+go mod init github.com/m
+go get -d -v github.com/google/codesearch@v1.2.0
+mkdir -p $prefix/bin
+go build -o $prefix/bin/csearch github.com/google/codesearch/cmd/csearch
+go build -o $prefix/bin/cindex github.com/google/codesearch/cmd/cindex
+exit
```

Comment:
In the end the result is the same, but it's better to use the most specific [environment variable](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1) when possible: `${bindir}` is the directory where the executables go:
```suggestion
mkdir -p "${bindir}"
go build -o "${bindir}/csearch${exeext}" github.com/google/codesearch/cmd/csearch
go build -o "${bindir}/cindex${exeext}" github.com/google/codesearch/cmd/cindex
```
I've also used `${exeext}` to adjust the extension of the executable on Windows